src/components/Pagination.js

```diff
@@ -46,8 +46,8 @@ const Pagination = ({ index: currentPageIndex, totalPages }) => {
       {currentPageIndex !== 0 ? (
         <Link
           css={itemStyles}
-          to={prevPageNumber === 0 ? '/' : `/page-${prevPageNumber}`}
-        >
+					to={prevPageNumber > 1 ? `/page-${prevPageNumber}` : `/`}
+				>
```

Comment:
Good catch. The formatting is a little off. Would you mind running `npm run format` and committing the update?

Reply:
Ok, this should clean up the formatting.

Reply:
Thank you. LGTM!